presto-manager-agent/src/main/java/com/teradata/prestomanager/agent/PrestoUpgrader.java

```diff
@@ -81,9 +84,10 @@ private static void upgradeUsingRpm(String pathToRpm, boolean disableDependencyC
         if (disableDependencyChecking) {
             nodeps = "--nodeps";
         }
-        String upgradeRpm = format("sudo rpm -U %s %s", nodeps, pathToRpm);
+        String upgradeRpm = format("sudo rpm -Uv %s %s", nodeps, pathToRpm);
```

Comment:
I am not sure if you should use -v mode. As you are feeding everything in the inputstream to log. Keep it precise should be the goal